projects/gloo/cli/pkg/cmd/gatewayapi/convert/domain/gatewayoutput.go

```diff
@@ -1,237 +1,98 @@
-	}
-	g.ListenerSets[l.NameIndex()] = l
-}
-
-func (g *GatewayAPICache) HasItems() bool {
-	if len(g.HTTPRoutes) > 0 {
-		return true
-	}
-	if len(g.RouteOptions) > 0 {
-		return true
-	}
-	if len(g.VirtualHostOptions) > 0 {
-		return true
-	}
-	if len(g.Upstreams) > 0 {
-		return true
-	}
-	if len(g.AuthConfigs) > 0 {
-		return true
-	}
-	if len(g.ListenerSets) > 0 {
-		return true
-	}
-	if len(g.HTTPListenerOptions) > 0 {
-		return true
-	}
-	if len(g.ListenerOptions) > 0 {
-		return true
-	}
-	if len(g.DirectResponses) > 0 {
-		return true
-	}
-	// if there are only yaml objects then skip because we didnt change anything in the file
-
-	return false
-}
-
-func (g *GatewayAPICache) ToString() (string, error) {
-	output := ""
-
-	for _, y := range g.YamlObjects {
-		output += "\n---\n" + y.Yaml + "\n"
-	}
-
-	for _, op := range g.Upstreams {
-		m, err := yaml.Marshal(op.Upstream)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-
-	for _, op := range g.HTTPRoutes {
-		m, err := yaml.Marshal(op.HTTPRoute)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-	for _, op := range g.RouteOptions {
-		m, err := yaml.Marshal(op.RouteOption)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-	for _, op := range g.AuthConfigs {
-		m, err := yaml.Marshal(op.AuthConfig)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-
-	for _, op := range g.VirtualHostOptions {
-		m, err := yaml.Marshal(op.VirtualHostOption)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-
-	for _, op := range g.ListenerOptions {
-		m, err := yaml.Marshal(op.ListenerOption)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-	for _, op := range g.HTTPListenerOptions {
-		m, err := yaml.Marshal(op.HttpListenerOption)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-	for _, op := range g.ListenerSets {
-		m, err := yaml.Marshal(op.XListenerSet)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-	for _, op := range g.DirectResponses {
-		m, err := yaml.Marshal(op.DirectResponse)
-		if err != nil {
-			return "", err
-		}
-
-		output += "\n---\n" + string(m)
-	}
-
-	// need to remove a few values
-	//  creationTimestamp: null
-	// status: {}
-	// status:
-	// parents: null
-	output = strings.ReplaceAll(output, "  creationTimestamp: null\n", "")
-	output = strings.ReplaceAll(output, "status:\n", "")
-	output = strings.ReplaceAll(output, "parents: null\n", "")
-	output = strings.ReplaceAll(output, "status: {}\n", "")
-	output = strings.ReplaceAll(output, "\n\n\n", "\n")
-	output = strings.ReplaceAll(output, "\n\n", "\n")
-	output = strings.ReplaceAll(output, "spec: {}\n", "")
-
-	// TODO remove leading and trailing ---
-	// log.Printf("%s", output)
-	return output, nil
-}
+//
+//type GatewayAPICache struct {
```

Comment:
Should we just delete this code? Or is there a particular reason we want to leave it commented out?